src/sdgio-edit.owl

```diff
@@ -3293,6 +3307,45 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020299> "increased capacity for innovation in partner development organizations and in poor and vulnerable communities"@en)
 ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020237> <http://purl.unep.org/sdg/SDGIO_00020299>)
 
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020302> (adaptation to and mitigation of climate change through agricultural research)
```

Comment:
"adapt to and mitigate..."

Reply:
I do that, but for the other DOs we followed the noun style (as opposed to the verb style). For example, we wrote 'improvement of food' rather than 'improve food'. Either way works for me. I just wonder whether there is a consistency requirement.

Reply:
We should use the active forms that sound more like objectives (e.g. improve, enhance, etc)
Keep this PR focused on the CCOs, but create an issue to fix this globally.

Reply:
I created issue #166 for this.